src/include/homeobject/blob_manager.hpp

```diff
@@ -9,17 +9,17 @@
 
 namespace homeobject {
 
-ENUM(BlobError, uint16_t, UNKNOWN = 1, TIMEOUT, INVALID_ARG, NOT_LEADER, UNKNOWN_SHARD, UNKNOWN_BLOB,
-     CHECKSUM_MISMATCH);
+ENUM(BlobError, uint16_t, UNKNOWN = 1, TIMEOUT, INVALID_ARG, NOT_LEADER, UNKNOWN_SHARD, UNKNOWN_BLOB, CHECKSUM_MISMATCH,
+     UNKNOWN_PG, PG_NOT_READY, READ_FAILED, INDEX_ERROR);
```

Comment:
Remove `PG_NOT_READY`; it's unused but will have to be handled in any `switch` statement on this type.